CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing
+First, and most importantly, thanks!
+
+## Running locally
+1. Install the requirements 
+   - Java 11
+   - SBT
+   - Docker
+2. Create configuration
+Create a configuration file at `~/.gu/riff-raff.conf`. At a minimum it looks like this,
+replacing placeholders with appropriate values:
```

Comment:
Would it be worth suggesting how to find out what appropriate values would be? I'm not sure myself!